pom.xml

```diff
@@ -56,7 +56,9 @@
       <module>rhq-plugin</module>
       <module>osgi</module>
       <module>spring/spring</module>
-      <module>spring/spring4</module>
```

Comment:
Some inconsistency: spring/spring4/pom.xml is never referenced directly by any reactor and therefore it doesn't get "installed" into the repository